v2/devices/merlin.yml

```diff
@@ -0,0 +1,163 @@
+name: "Xiaomi Redmi Note 9"
+codename: "merlin"
+aliases: []
+formfactor: "phone"
+doppelgangers: []
+user_actions:
+  recovery:
+    title: "Reboot to Recovery"
+    description: "With the device powered off, hold Volume Up + Power."
+    image: "phone_power_up"
+    button: true
+  bootloader:
+    title: "Reboot to Bootloader"
+    description: "With the device powered off, hold Volume Down + Power."
+    image: "phone_power_down"
+    button: true
+  boot:
+    title: "Boot the device"
+    description: "Power on the device."
+    image: "phone_power_up"
+    button: true
```

Comment:
unused `user_action`

Reply:
Fixed. On related note, there currently seems to be no way to reboot from fastboot to recovery? (`fastboot reboot recovery` command)

Reply:
thanks, will investigate that